### DISCUSSION TITLE
Add cursor folder to the skin editor

### DISCUSSION THREAD

**Discussion by reo-the-enby** [ppy/osu#22938] [Ideas]
Some skins have folders in them with different cursors so you can swap them out or swap out other elements (top left)
Would be handy if you could find this folder somewhere in the skin editor and swap cursors out.

![bild](https://user-images.githubusercontent.com/81715029/227019914-ddfbcd58-a69f-46e4-a1c0-d67e0bc1c558.png)

**Comment by Bruno5430** [comment]
Do those skins actually have different designs included or its just the same cursor with different colors?
In that case would be cool to have a value in `skin.ini` or similar to change the color of the cursor and to be editable from the skin editor

**Reply by Bruno5430** [reply]
you could also consider another idea like using multiple skins at the same time, mixing different stuff like hitsounds from one skin and the cursor from another one

**Comment by Rekunan** [comment]
To generalize the idea (if it gets implemented) would be to have a way of storing alternative elements in the game to easily swap to, perhaps just as another section or tab in the skin editor.

To provide a use-case, I put on 300 images whenever I play in public to help with understanding the gameplay, but at home I make them blank, and to do this I just have 2 skins of the same skin where the only difference is the 300 image, and I swap between them.
